### ISSUE TITLE
BTT GTR V1.0 Cant compile 

### ISSUE THREAD

**Issue by Ski3d** [MarlinFirmware/Marlin#17009]
### Bug Description

Cant compile GTR V1.0 Using Bugfix or newest release

### My Configurations

Have only Config the settings below so is basically vanilla marlin to see if i can even get it to compile
```cpp
/**
 * Select the serial port on the board to use for communication with the host.
 * This allows the connection of wireless adapters (for instance) to non-default port pins.
 * Note: The first serial port (-1 or 0) will always be used by the Arduino bootloader.
 *
 * :[-1, 0, 1, 2, 3, 4, 5, 6, 7]
 */
#define SERIAL_PORT -1

/**
 * Select a secondary serial port on the board to use for communication with the host.
 * This allows the connection of wireless adapters (for instance) to non-default port pins.
 * Serial port -1 is the USB emulated serial port, if available.
 *
 * :[-1, 0, 1, 2, 3, 4, 5, 6, 7]
 */
#define SERIAL_PORT_2 3

/**
 * This setting determines the communication speed of the printer.
 *
 * 250000 works in most cases, but you might try a lower speed if
 * you commonly experience drop-outs during host printing.
 * You may try up to 1000000 to speed up SD file transfer.
 *
 * :[2400, 9600, 19200, 38400, 57600, 115200, 250000, 500000, 1000000]
 */
#define BAUDRATE 115200

// Enable the Bluetooth serial interface on AT90USB devices
//#define BLUETOOTH

// Choose the name from boards.h that matches your setup
#ifndef MOTHERBOARD
  #define MOTHERBOARD BOARD_BTT_GTR_V1_0
#endif
```

### Steps to Reproduce

1. Set setting above 
2. Build using Visual studio code/PlatformIO using the Env:BIGTREE_GTR_V1_0 in project tasks

**Expected behavior:** Compile 

**Actual behavior:** Failed
 

#### Additional Information
And sorry if its just a dumb mistake but i can compile an SKR 1.3, SKR mini v1,0 and stock ender boards with no problems.

![Capture1](https://user-images.githubusercontent.com/61573573/75512009-1178a900-59be-11ea-963d-7230cfcbbe1d.PNG)
![Capture2](https://user-images.githubusercontent.com/61573573/75512282-e17dd580-59be-11ea-8ce3-c09392a4345d.PNG)
![Capture3](https://user-images.githubusercontent.com/61573573/75512283-e2166c00-59be-11ea-92f5-49cb8eb96f44.PNG)
![Capture4](https://user-images.githubusercontent.com/61573573/75512284-e2166c00-59be-11ea-8956-302e1e23c913.PNG)
![Capture5](https://user-images.githubusercontent.com/61573573/75512285-e2af0280-59be-11ea-9136-cf200f13486e.PNG)
![Capture6](https://user-images.githubusercontent.com/61573573/75512286-e2af0280-59be-11ea-8e43-0c25a25fc256.PNG)
![Capture7](https://user-images.githubusercontent.com/61573573/75512323-083c0c00-59bf-11ea-8f57-443db56ada5b.PNG)
[Configuration.zip](https://github.com/MarlinFirmware/Marlin/files/4269174/Configuration.zip)

**Comment by ellensp** [comment]
Using my best guess at your config (you should ALWAYS include configuration files, then we could just CTRL-A, CTRL-C, CTRL-V and be guaranteed to have your configuration)
This complies fine. Lots of ugly warning, but it does compile.
It looks like it is failing on the final link. 
This is is probably the windows command line limit. 
Delete the directory Marlin/src/lcd/extensible_ui  (as you not using a display) and try again. if it now compiles... Its the above issue.

**Comment by ellensp** [comment]
And for future reference please post text logs!, how can we search for a error in a bunch of images?

**Comment by Ski3d** [comment]
Well it did not work like you said, but i found out that if i removed the arrow (>) from this line it compiles just fine..
Old: framework-arduinoststm32@>=3.10700.191028
New:framework-arduinoststm32@=3.10700.191028
![arr](https://user-images.githubusercontent.com/61573573/75593426-539ffa00-5a53-11ea-8ac2-15c45c3852e3.PNG)

**Comment by Ski3d** [comment]
Can this be added to the new bug fix? i am not the only one have this same problem and would probably help others.

**Comment by thinkyhead** [comment]
Currently the line is this:
```ini
platform_packages = framework-arduinoststm32@>=3.107,<4
```

**Comment by Ski3d** [comment]
By removing the > in that line, it will compile with no fails.

**Comment by Ski3d** [comment]
So it has changed since my last download of the bugfix. So we shouldn't see the same problem as i have?.  When I posted2 days ago ish, I was using the latest bugfix

**Comment by NAPCAL** [comment]
Too deep in the folders. Move the main Marlin into the root. The compiler has issues with long paths ends up being truncated but platformio reports back the full path.

Here is my complies for the GTR & Pro, no issues and all are in H:\ 
![image](https://user-images.githubusercontent.com/47440988/79045368-56ece080-7bd0-11ea-8a83-6c83ea5ea5c2.png)

If this doesn't fix your issue then please zip up these files and provide them.
\platformio.ini
\Marlin\Configuration.h
\Marlin\Configuration_adv.h
\Marlin\src\pins\stm32f4\pins_BTT_SKR_PRO_V1_1.h
And any other source files you have made changes.

**Comment by NAPCAL** [comment]
Also, the Bugfix is updated nightly so issues might show up that didn't yesterday.

**Comment by b3nnyb3n** [comment]
![gtrVskr](https://user-images.githubusercontent.com/51232272/82775262-90c92c00-9e8a-11ea-9fcc-cc34788bff1f.png)
Hi guys Im having issues too. Would love to know how youve gotten so far when.... I have compared Official  Marlin / BTT marlin / & BTT gtr Marlin builds. They all point to the Pins file for SKR PRO. Pins totally miss match.

I uploaded a build to my gtr sd card and usb arent fuctioning now. (also fresh outta noob school have alot to learn)
